packages/central-server/src/dhis/DhisChangeValidator.js

```diff
@@ -20,18 +21,20 @@ export class DhisChangeValidator extends ChangeValidator {
         `,
       )
     ).map(r => r.user_id);
+
     const validSurveyResponseIds = [];
     const batchSize = this.models.database.maxBindingsPerQuery - nonPublicDemoLandUsers.length;
     for (let i = 0; i < surveyResponseIds.length; i += batchSize) {
       const batchOfSurveyResponseIds = surveyResponseIds.slice(i, i + batchSize);
+
       const batchOfSurveyResponses = await this.models.database.executeSql(
         `
           SELECT DISTINCT survey_response.id as id
           FROM survey_response
           JOIN survey ON survey_response.survey_id = survey.id
           JOIN data_group ON data_group.id = survey.data_group_id
           JOIN entity ON survey_response.entity_id = entity.id
-          AND data_group.service_type = 'dhis'
+          where data_group.service_type = 'dhis'
```

Comment:
Oh this was all part of the join previously?? Huh... I think this change is safe and definitely easier to reason about...

Reply:
Yep it was, my assumption is it should be filtering by dhis service types so I think it makes sense?

Reply:
So it will still filter actually when you add `AND` clauses in the join, but I'm not certain if the filtering would resolve in the same way.

For instance:
```
SELECT * FROM ancestor_descendant_relation adr
JOIN entity e ON adr.entity_id = e.id AND e.type = 'country';

# will resolve to the same set as:
SELECT * FROM ancestor_descendant_relation adr
JOIN entity e ON adr.entity_id = e.id 
WHERE e.type = 'country';
```

Could I ask you to just double check this one actually :pray:

Reply:
I just reverted it to the old way, just to make sure there are no regressions